### PR TITLE
Put popular_artists module first for logged-out users

### PR DESCRIPTION
**Pull request by bhoggard**
I also added a fix for the setup script's `nvm` check.

Related issue: https://github.com/artsy/collector-experience/issues/362

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -17,7 +17,7 @@ printf "[1/${STEPS}]  checking dependencies"
 actual_node_version=$(node --version)
 expected_node_version=$(cat .nvmrc)
 
-if ! command -v nvm > /dev/null; then
+if [ -d "~/.nvm" ]; then
```
Comment (jonallured) [inline bin/setup]: Nice, thanks @bhoggard!! ❤️

**Comment by bhoggard** [comment]
This is needed, as the auction still is at the top of the page in Force, even when we pass in an `order` parameter.

**Comment by xtina-starr** [comment]
👍

**Comment by alloy** [comment]
Once this is in, it would be nice if you could check why the specified order is not working as expected in this case.

**Pull request merged**